include/circt/Dialect/RTL/Combinatorial.td

```diff
@@ -63,6 +63,27 @@ class UTBinRTLOp<string mnemonic, list<OpTrait> traits = []> :
   }];
 }
 
+// Base class for variadic operators.
+class VariRTLOp<string mnemonic, list<OpTrait> traits = []> :
```

Comment:
nit: The formatting on this seems a bit weird. Does Circt have a specific style guide for tablegen? Genuinely curious.

Reply:
I don't know about CIRCT style guide for tablegen. How would you change it?

Reply:
We don't have a bespoke style guide here, we are trying to follow MLIR style for "all the things"

Reply:
I think that more typical style would be to put the colon on the next line before `RTLOp<`

Reply:
The lack of decent autoformatting for tablegen is something I've always been annoyed by, but never quite bothered trying to fix.

Reply:
Yeah, agreed, it would be really nice.